site-modules/gplt_runner_setup/manifests/core.pp

```diff
@@ -46,7 +46,22 @@
     package => "java-1.8.0-openjdk-devel",
   }
 
-  include nginx
+  class { "nginx":
+    manage_repo    => true,
+    package_source => "nginx-stable",
+    confd_only     => true,
+  }
+
+  nginx::resource::server{ "default":
+    www_root => "/usr/share/nginx/html/users",
```

Comment:
Should this resource just be moved to the user setup, set to  `"/home/${test_user}/gatling"`, and do away with link wrangling?

Reply:
So the idea is there could be multiple users setup on the instance.  This creates the base users dir to hold the links.